compiler/sem/transf.nim

```diff
@@ -174,6 +188,31 @@ proc freshVar(c: PTransf; v: PSym): PNode =
     newVar.owner = owner
     result = newSymNode(newVar)
 
+proc transformDefSym(c: PTransf, n: PNode): PNode =
```

Comment:
It's probably worth marking this as `{.deprecated: "workaround sem not sanitizing AST".}`